src/App.tsx

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<number>(5);
+  const [currentPage, setCurrentPage] = useState<number>(1);
+
+  const pagiTotal = 42;
```

Comment:
move outside the component
```suggestion
  const totalPages = 42;
```